src/test/groovy/graphql/validation/rules/SingleSubscriptionFieldOnlyTest.groovy

```diff
@@ -0,0 +1,66 @@
+package graphql.validation.rules
+
+import graphql.language.SourceLocation
+import graphql.parser.Parser
+import graphql.validation.SpecValidationSchema
+import graphql.validation.ValidationError
+import graphql.validation.ValidationErrorType
+import graphql.validation.Validator
+import spock.lang.Specification
+
+import static graphql.validation.rules.SingleSubscriptionFieldOnly.moreThanOneSubscriptionMessage
+
+class SingleSubscriptionFieldOnlyTest extends Specification {
+
+    def '5.2.3.1 Subscription operations must have exactly one root field.'() {
```

Comment:
can we remove the spec section references? It has not really payed of over the long time and it is also not guaranteed to be stable.